pkg/proxy/proxy.go

```diff
@@ -30,8 +32,13 @@ const (
 	maxRequestBodyBytes  = 512
 	vinLength            = 17
 	proxyProtocolVersion = "tesla-http-proxy/1.1.0"
+	MaxResponseLength    = 10000000
+	MaxRetries           = 3
```

Comment:
The only retried error is a 421. Do we expect more than one of those consecutively?

Reply:
No. I can lower it to 2.